### DISCUSSION TITLE
Migration from Minio/S3 to local storage

### DISCUSSION THREAD

**Discussion by piranha771** [outline/outline#6082] [Self Hosting] [answered]
Is there a migration path for Minio/S3 to the local storage for existing outline installations?

**Comment by Limezy** [comment]
Do you mean a procedure to follow ?

**Reply by piranha771** [reply]
Exactly. I have used Minio as the storage before and would like to use local storage now.

**Reply by tommoor** [reply, marked as answer]
There isn't really a works-for-everyone guide but you just need to put the files from Minio somewhere on your disk keeping the same structure and then point `FILE_STORAGE_LOCAL_ROOT_DIR` at that directory. You may hit permissions issues, in which case see the notes in the [documentation](https://docs.getoutline.com/s/hosting/doc/file-storage-N4M0T6Ypu7#h-file-system-folder).

**Reply by beastarman74** [reply]
Will this work when looking to migrate from AWS S3 storage to local storage?

**Reply by tommoor** [reply]
If you re-create the same directory structure locally that is in the AWS buckets then I don't see any reason why not!
